docs/index.md

```diff
@@ -74,49 +71,104 @@ Some basic charts are already included in Zeppelin. Visualizations are not limit
   </div>
 </div>
 
-#### Pivot chart
+### Pivot chart
 
-With simple drag and drop Zeppelin aggregates the values and display them in pivot chart. You can easily create chart with multiple aggregated values including sum, count, average, min, max.
+Apache Zeppelin aggregates values and displays them in pivot chart with simple drag and drop. You can easily create chart with multiple aggregated values including sum, count, average, min, max.
 
 <div class="row">
   <div class="col-md-8">
     <img class="img-responsive" src="/assets/themes/zeppelin/img/screenshots/pivot.png" />
   </div>
 </div>
-Learn more about Zeppelin's Display system. ( [text](./displaysystem/display.html), [html](./displaysystem/display.html#html), [table](./displaysystem/table.html), [angular](./displaysystem/angular.html) )
+
+Learn more about [display systems](#display-system) in Apache Zeppelin.
 
 <br />
-### Dynamic forms
+## Dynamic forms
 
-Zeppelin can dynamically create some input forms into your notebook.
+You can dynamically create some input forms in your notebook.
```

Comment:
Generally, previous writing style of "Zeppelin can ..." is prefered to the one that is dominant in this PR, like "You can ...", "Your notebooks ..." - first one feels more like a documentation about the project.

What do you think, is it possible\make sense to adjust it here?